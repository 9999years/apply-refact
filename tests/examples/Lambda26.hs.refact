[("tests/examples/Lambda26.hs:1:7-15: Warning: Avoid lambda\nFound:\n  \\ x -> a x\nPerhaps:\n  a\n",[]),("tests/examples/Lambda26.hs:1:17-21: Suggestion: Redundant where\nFound:\n  where\nPerhaps you should remove it.\n",[])]
