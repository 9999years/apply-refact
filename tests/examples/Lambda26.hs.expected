yes = \x -> a x where
